File: src/main/java/com/example/moexbondservice/client/GovernmentBondClient.java
package com.example.moexbondservice.client;

import org.springframework.cloud.openfeign.FeignClient;
import org.springframework.web.bind.annotation.GetMapping;

@FeignClient(name = "governmentBonds", url = "${moex.bonds.government.url}",  configuration = FeignConfig.class)
public interface GovernmentBondClient {

    @GetMapping
    String getBondsFromMoex();
}

File: src/main/java/com/example/moexbondservice/client/GovernmentBondClient.kt
package com.example.moexbondservice.client

import org.springframework.cloud.openfeign.FeignClient
import org.springframework.web.bind.annotation.GetMapping

@FeignClient(name = "governmentBonds", url = "\${moex.bonds.government.url}", configuration = [FeignConfig::class])
interface GovernmentBondClient {
    @GetMapping
    fun getBondsFromMoex(): String?
}
